### PR TITLE
Add jsonnet

### PR DESCRIPTION
**Pull request by nelson-liu**
Adding jsonnet (https://github.com/google/jsonnet) to conda-forge.

### CODE REVIEW

**Comment by nelson-liu** [comment]
Hmm, a bit confused by what's going on here. It seems like it isn't picking up the proper includes flags from the makefile (https://github.com/google/jsonnet/blob/v0.10.0/Makefile#L32-L37)?

The `setup.py` file essentially calls `make` (https://github.com/google/jsonnet/blob/v0.10.0/setup.py#L49-L55) . What's interesting is that if i run the `build_conda.sh` script generated by `conda build` on my local machine, it fails with the same error, but if I just run `pip install . --no-deps -vv`, it works fine. Perhaps the flags are being overwritten somehow by the conda build process?

**Comment by nelson-liu** [comment]
@conda-forge/staged-recipes looks like this is ready to go --- let me know if this is not the preferred way to override compiler flags.

File: recipes/jsonnet/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export OPT="$OPT -O3"
+export CXXFLAGS="$CXXFLAGS -g $(OPT) -Wall -Wextra -Woverloaded-virtual -pedantic -std=c++0x -fPIC -Iinclude -Ithird_party/md5"
+export CFLAGS="$CFLAGS -g $(OPT) -Wall -Wextra -pedantic -std=c99 -fPIC -Iinclude"
+export MAKEDEPENDFLAGS="$MAKEDEPENDFLAGS -Iinclude -Ithird_party/md5"
+export SHARED_LDFLAGS="$SHARED_LDFLAGS -shared"
+
+pip install . --no-deps -vv
```
Comment (scopatz) [inline recipes/jsonnet/build.sh]: I think this should be `$PYTHON -m pip install . --no-deps -vv` in order to get the correct Python executable

File: recipes/jsonnet/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python={{ python }}
```
Comment (scopatz) [inline recipes/jsonnet/meta.yaml]: There needs to be a space before the equals sign, e.g. `- python ={{ python }}`

**Comment by nelson-liu** [comment]
@scopatz thanks for taking a look, i've made the changes above.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**